setup.py

```diff
@@ -12,7 +12,7 @@
     include_dirs  = ['http-parser', '/usr/include/libev'],
     define_macros = [('WANT_SENDFILE', '1'),
                      ('WANT_SIGINT_HANDLING', '1')],
-    extra_compile_args = ['-std=c99', '-fno-strict-aliasing', '-fcommon',
+    extra_compile_args = ['-std=c11', '-fno-strict-aliasing', '-fcommon',
```

Comment:
It looks like if the "-std=c99/c11" is removed completely then it also builds fine without any warnings. So I added a new patch that removed "-std=c99/c11" from setup.py.